docs/guides/beta-terms.md

```diff
@@ -0,0 +1,31 @@
+---
+id: beta-terms
+title: Beta terms
+description: "Learn how to launch the beta terms agreement as an Admin user during the pre-release phase of software development. "
+keywords: [beta-terms, beta, admin, pre-release]
+---
+
+:::note
+Camunda beta terms currently only apply to Enterprise SaaS subscriptions. For all other offers (e.g. Professional Plan, Free-Trial or Tier-Plan) users need not accept special beta terms; it is sufficient to agree to our standard Terms & Conditions which also contain a provision on beta terms.
+:::
+
+A beta test is the second phase of software testing in which a sampling of the intended audience can test the product. When Camunda refers to **beta**, we reference anything pre-release, such as any version of the software designated as alpha, beta, limited release, developer preview, non-production, or evaluation.
+
+Beta terms typically refer to the specific terms and conditions that govern the use and testing of this software during its beta phase. These terms outline the rights, responsibilities, and limitations of both the software provider (Camunda) and the users (beta testers or early adopters) during the testing and evaluation period.
+
+Beta terms help protect Camunda´s interests (such as protecting our intellectual property, disclaiming warranties, or limiting our liability for any issues or damages that may arise during the beta phase), manage user expectations, encourage active participation and feedback, and ensure legal compliance during the pre-release phase of software development.
+
+:::note
+Enabling beta is limited to [admin users](../components/console/manage-organization/manage-users.md) of Camunda products.
+:::
+
+## How to accept beta terms
+
+To accept beta terms for Camunda products, follow the steps below:
+
+1. Log in to Camunda Console and click the **Organization** tab to view the overview for **Organization Management**.
+   1. If the **Pre-Release Features** is marked as **Disabled**, you do not currently have admin access rights to the Camunda beta features. Visit our documentation on [managing users](../components/console/manage-organization/manage-users.md) to learn more.
```

Comment:
In the latest Iteration the Opt-In Button moved to the `Settings` tab in `Organization Management` 
The terms are now displayed inside a modal and the user can then toggle individual features. 
![image](https://github.com/camunda/camunda-docs/assets/3696742/5b468fa6-556d-4aa9-94b7-793a0d8047eb)
![image](https://github.com/camunda/camunda-docs/assets/3696742/5793f080-6d96-4f23-b352-3cf4aa7a86fb)
![image](https://github.com/camunda/camunda-docs/assets/3696742/47149841-6c29-4f43-96cf-ec050d096702)